src/web/tools/PointerEventManager.ts

```diff
@@ -234,11 +238,13 @@ export default class PointerEventManager extends EventManager<HTMLElement> {
   }
 
   protected mapEvent(event: PointerEvent, eventType: EventTypes): AdaptedEvent {
+    const { scaleX, scaleY } = calculateViewScale(this.view);
+
     return {
       x: event.clientX,
       y: event.clientY,
-      offsetX: event.offsetX,
-      offsetY: event.offsetY,
+      offsetX: event.offsetX / scaleX,
+      offsetY: event.offsetY / scaleY,
```

Comment:
Wouldn't this also affect other places? I'm not sure in which cases relative coords are used on web.

Reply:
Right now it is used only in `PointerTracker` to get relative coords. Since relative coords have just been introduced, this change shouldn't affect anything.